tests/karma/unit/crmCaseTypeSpec.js

```diff
@@ -15,22 +23,51 @@ describe('crmCaseType', function() {
     inject(function(crmJsonComparator) {
       crmJsonComparator.register(jasmine);
     });
+
+    caseTypeCategories = {
```

Comment:
Move this to a separate function called `caseTypeCategoriesSampleData`(or something similar). And call that in line 70.